src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,31 +48,61 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+        boolean answer = false;
```

Comment:
Complex is better than complicated
`return getNumberOfOccurrences(array, value) > 0`